src/Hazelcast.Net/DistributedObjects/FlakeIdGeneratorOptions.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading;
+
+namespace Hazelcast.DistributedObjects
+{
+    /// <summary>
+    /// Represents options for <see cref="IFlakeIdGenerator"/>.
+    /// </summary>
+    public class FlakeIdGeneratorOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FlakeIdGeneratorOptions"/> class.
+        /// </summary>
+        public FlakeIdGeneratorOptions()
+        { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FlakeIdGeneratorOptions"/> class.
+        /// </summary>
+        /// <param name="other">Instance to copy option values from.</param>
+        private FlakeIdGeneratorOptions(FlakeIdGeneratorOptions other)
+        {
+            PrefetchCount = other.PrefetchCount;
+            PrefetchValidityPeriod = other.PrefetchValidityPeriod;
+        }
+
+        /// <summary>
+        /// Gets the number of identifiers that are pre-fetched when a new id is requested from the cluster.
+        /// </summary>
+        /// <remarks>
+        /// <para>Allowed values are between <c>1</c> and <c>100000</c> inclusive, default value is <c>100</c>.</para>
+        /// </remarks>
+        public int PrefetchCount { get; set; } = 100;
+
+        /// <summary>
+        /// Gets the
```

Comment:
Added by mistake?

Reply:
Silly me. Wanted to cleanup the summary to use the more-or-less standard "Gets the..." pattern and did not fully do it, hence the FIXME comment at the end.

Reply:
Adjuster summary.